FirebaseCore/CHANGELOG.md

```diff
@@ -1,9 +1,22 @@
 # Firebase 10.0.0
 - [changed] **Breaking change**: Firebase's minimum supported versions have
   updated for the following platforms:
-  - iOS 11.0 _(previously 10.0)_
-  - macOS 10.13 _(previously 10.12)_
-  - tvOS 11.0 _(previously 10.0)_
+  - If using **CocoaPods**:
+    - | Platform  | Firebase 9 | Firebase 10 |
+      | ------------- | ------------- | ------------- |
+      | iOS  | 10.0  | **11.0**  |
+      | tvOS  | 10.0  | **11.0**  |
+      | macOS  | 10.12  | **10.13**  |
+      | watchOS  | 6.0  | 6.0  |
+  - If using **Swift Package Manager**:
+    - | Platform  | Firebase 9 | Firebase 10 |
+      | ------------- | ------------- | ------------- |
+      | iOS  | 11.0  | 11.0  |
+      | tvOS  | 11.0  | 11.0  |
+      | macOS  | 10.12  | **10.13**  |
+      | watchOS  | 7.0  | 7.0  |
+  - If using the **Zip** distribution, there are **no changes** to the minimum supported versions.
```

Comment:
Thanks! Add a table instead, so it's clear what the minimum versions are. (matching Firebase 9 for iOS, tvOS, and macOS, n/a for watchOS

Reply:
Yep, added a tables to show

Reply:
PTAL [be6a8cb](https://github.com/firebase/firebase-ios-sdk/pull/10233/commits/be6a8cb0a6eb967666fcd8079250d3216fa0f1e6)

Reply:
LGTM, also I'd be ok to combine Carthage and zip.

Reply:
Good idea, thanks. Done [3476119](https://github.com/firebase/firebase-ios-sdk/pull/10233/commits/3476119ea6187dea60793eddcac83a67e439e575)